src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -271,7 +272,7 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
     }
 
     private getLocationItems(buckets: IBucket[]): IBucketItem[] {
-        const locationItems: IBucketItem[] = getPreferredBucketItems(
+        const locationItems: IBucketItem[] = getItemsFromBuckets(
```

Comment:
Can you please briefly explain this change?

Reply:
if the VIEW bucket have 2 items with type ATTRIBUTE and DATE,
getPreferredBucketItems([VIEW], [BucketNames.VIEW], [ATTRIBUTE]) -> []
But getItemsFromBuckets([VIEW], [BucketNames.VIEW], [ATTRIBUTE]) -> [] -> [ATTRIBUTE]

because of condition `every` [here](https://github.com/gooddata/gooddata-react-components/blob/develop/src/internal/utils/bucketHelper.ts#L357)